parlai/core/torch_generator_agent.py

```diff
@@ -433,7 +440,10 @@ def _init_cuda_buffer(self, batchsize, maxlen, force=False):
         if self.use_cuda and (force or not hasattr(self, 'buffer_initialized')):
             try:
                 loss = self.compute_loss(self._dummy_batch(batchsize, maxlen))
-                loss.backward()
+                if self.fp16:
```

Comment:
@emilydinan what's your opinion? Is it worth building a `def backward(self, loss)` into TorchAgent so that this is abstracted?

Reply:
given that this shows up a few times, it might be worth it

Reply:
Sorry I did not see this comment until now -- yes seems fine to me.